set.mm

```diff
@@ -439147,6 +439147,18 @@ orthogonal vectors (i.e. whose inner product is 0) is the sum of the
 htmldef "RRhat" as "RRhat";
   althtmldef "RRhat" as "&#8477;&#x302;";
   latexdef "RRhat" as "\hat{\mathbb{R}}";
+htmldef "NNbar" as "NNbar";
```

Comment:
Is this fixed to this set https://en.wikipedia.org/wiki/Extended_natural_numbers?

Reply:
Yes, it's only a set; I should add its definition as ` |- NNbar = ( NN0 u. pinfty ) `.

Reply:
(added the three definitions of NNbar, ZZbar, ZZhat)